apps/web-giddh/src/app/welcome/welcome.component.html

```diff
@@ -163,6 +153,11 @@ <h2>Welcome To Giddh! </h2>
                             </div>
                         </div>
                     </ng-container>
+                    <!-- Show the state drop down for countries UK, US and Nepal for which we don't support taxation for now
+                            and business type is 'Others' than 'Registered' or 'Unregistered' -->
+                    <ng-container
+                        *ngTemplateOutlet="(!formFields['taxName'] && businessTypeList?.length === 1 && businessTypeList[0].label === 'Others') && !isWarehouse ? stateTemplate : ''">
```

Comment:
Add nulity check for businessTypeList[0]

Reply:
No need as I've checked for businessTypeList?.length === 1 in && condition